verifiers & apps.md

```diff
@@ -0,0 +1,48 @@
+CommonPass                 | yes (read and produce) | yes  | yes? | yes | no? | periodic/manual | no (wallet) | [iOS](https://apps.apple.com/us/app/commonpass/id1548682047), android?, [developer](https://commonpass.org/)
+New MITRE verifier web app | ?                      | ?    |?|?|?|? | yes?
+
+## Wallet or other holder apps
+|| QR code support | .smart-health-card support | API support | reads vaccinations | reads labs | time-delay to sync VCI issuer directory | links
+| --- | --- | --- | --- | --- | --- | --- | --- | 
+CommonPass                 | yes (read and produce) | yes  | yes? | yes | no? | periodic/manual | [iOS](https://apps.apple.com/us/app/commonpass/id1548682047), android?, [developer](https://commonpass.org/)
+CommonHealth               | (yes, read and produce)|      ||||| [android](https://play.google.com/store/apps/details?id=org.thecommonsproject.android.phr), [developer](https://www.commonhealth.org/)
+CLEAR Health Pass          | yes (read only)        | ?    | ? | yes | yes | ? |  [iOS](https://apps.apple.com/us/app/clear-fast-touchless-access/id1436333504), [android](https://play.google.com/store/apps/details?id=com.clearme.clearapp), [developer](https://www.clearme.com/healthpass)
+Apple Health               | yes (read and produce) | yes  | yes | yes | yes | periodic |  [developer](https://support.apple.com/en-us/HT212752)
+
+## Technical, developer tools
+
+### Validators
+SMART Health Cards validators are website or applications which read a SMART Health Card and identify any problems with its content, formatting, structure, etc. towards the goal of ensuring it can be read by verifier, wallets and other apps. 
+
+|| Usable by non-programmers? | Validates FHIR content? | links | 
+| ---- | -- | -- | --- | 
+| Health Cards Dev Tools | No. |Partial. | https://github.com/smart-on-fhir/health-cards-dev-tools | 
+| demo-portals.smarthealth.cards | Yes! | No. | https://demo-portals.smarthealth.cards/VerifierPortal.html | 
+| Health Cards Validation SDK | No. | No. | https://github.com/microsoft/health-cards-validation-SDK | 
+| HL7 FHIR Validator | No. | Only validates FHIR content | https://confluence.hl7.org/display/FHIR/Using+the+FHIR+Validator | 
+| Grahame's FHIR toolkit (in progress -- check back here!) | | | | 
+| Online Health Certificate Decoder | No. | No. | https://ehealth.vyncke.org/index.php |
+| Pathcheck Verifiable Credential Debugger | No. | No. | https://github.pathcheck.org/debug.html | 
+
+
+### Additional developer resources
+
+|| Description | Purpose | License | links |
+| ----- | ----- | ---- | -- | -- |
+| demo-portals.smarthealth.cards | Starting with a FHIR Bundle, generate a SMART Health Card | Educational and troubleshooting tool. | Unknown. | https://demo-portals.smarthealth.cards/DevPortal.html |
```

Comment:
This is available under MIT License -- see https://github.com/smart-on-fhir/health-cards-tests/blob/master/LICENSE\

```suggestion
| demo-portals.smarthealth.cards | Starting with a FHIR Bundle, generate a SMART Health Card | Educational and troubleshooting tool. | MIT | https://demo-portals.smarthealth.cards/DevPortal.html |
```